### ISSUE TITLE
Surface reflectance for L8

### ISSUE THREAD

**Issue by para2x** [midraed/water#35]
Hey man,
I was wondering if you guys have any plans for adding function for estimation surface reflectance for L8?
I'm using Water V:0.8 and it asks users to download surface reflectance from espa website.

**Comment by midraed** [comment]
Yes, we can do it. But the model used by ESPA is many times better than any
model we can implement inside the package. The surface reflectance has a
great impact in the land surface energy balance model.  So, I recommend
using the model from USGS. But we can also apply a similar approach as the
one already included for landsat 7.

Best,

2017-11-26 18:06 GMT+01:00 Hamze Dokoohaki <notifications@github.com>:

**Comment by para2x** [comment]
Thanks for response.
Have you guys developed a function inside the package to do the same for L8 or we need to get inspired bu the code for L7 and use the similar approach ?

**Comment by midraed** [comment]
I prefer to use the usgs.gov estimation. However we can do somethign similar to what is done on L7. 
If you want, you can compare the current model for L7 using L8 data and compare the result with the usgs estimation. Then, we can include it on water. Tell me if you are interested and I can work with you on this.
